community/wg-cnbi/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+
+for Thoth we use `podman run --rm -e WHAT -e GO111MODULE=on -e GOPROXY -v $(pwd):/go/src/app:Z golang:1.12 make -C /go/src/app generate`
+
+--->
+# CNBi Working Group
+
+The goal of this working group is to create a service that implements the backend side of the Custom Notebook Image (CNBi) MVP in the context of Open Data Hub (ODH).
+
+The [charter](charter.md) defines the scope and governance of the CNBi Working Group.
+
+## Stakeholder SIGs
+* SIG DevSecOps
+* SIG Stack Guidance
+* SIG User Experience
+
+## Meetings
+* Regular CNBi Meeting: [Wednesdays at 14:00 UTC](https://meet.google.com/cyo-imtt-twa) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=14:00&tz=UTC).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1NQJ2gv6lYTcgxFh2nTlVZpGVh1nb2C43yqPh7igvRRM/edit).
+
+## Organizers
+
+* Christoph Görn (**[@goern](https://github.com/goern)**), Red Hat
+
+## Contact
+- Slack: [#wg-custom-notebook-images](https://kubernetes.slack.com/messages/wg-custom-notebook-images)
```

Comment:
this seems to be wrong, either we fold this WG into the ODH Slack, or we keep it on the Thoth dev chat, I and .51 on ODH Slack. WDYT?